dex/testing/firo/harness.sh

```diff
@@ -1,4 +1,41 @@
 #!/usr/bin/env bash
+
+# This simnet harness sets up 4 Firo nodes and a set of herness controls
+# Each node has a prepared, encrypted, empty wallet
+
+# alpha/
+# ├── alpha.conf
+# └── regtest
+#     ├── wallet.dat
+
+# beta/
+# ├── beta.conf
+# └── regtest
+#     ├── wallet.dat
+
+# gamma/
+# ├── gamma.conf
+# └── regtest
+#     ├── wallet.dat
+
+# delta/
+# ├── delta.conf
+# └── regtest
+#     ├── wallet.dat
+
+# └── harness-ctl
+#     ├── alpha
+#     ├── beta
+#     ├── connect-alpha
+#     ├── delta
+#     ├── gamma
+#     ├── mine-alpha
+#     ├── mine-beta
+#     ├── quit
+#     ├── reorg
+#     ├── start-wallet
+#     └── stop-wallet
```

Comment:
Unnecessary IMO.  These are lines that will inevitably just become out of date.

Reply:
Removed, thanks